department-of-veterans-affairs/abd-vro
# added script to generate bip keystore/truststore related vault secrets

## What was the problem?


Associated tickets or Slack threads:
- #2293

## How does this fix it?[^1]
created `bip-certgen.sh` to generate keystore & truststore vault secrets 

## How to test this PR
- have k8s kube_config file ready and set locally and kubectl tool installed
- run `./scripts/bip-certgen.sh ` in terminal (clean up & delete files the script generates at the end)


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
